### PR TITLE
Create fortify.yml

### PR DESCRIPTION
**Pull request by miclosgyjf**

### CODE REVIEW

**Comment by miclosgyjf** [comment]
Clase

**Review by Elmago34 — APPROVED** [review]
Gracias

File: .github/workflows/fortify.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  schedule:
+    - cron: '38 1 * * 5'
+  workflow_dispatch:
+
+jobs:
+  Fortify-AST-Scan:
+    # Use the appropriate runner for building your source code. Ensure dev tools required to build your code are present and configured appropriately (MSBuild, Python, etc).
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
```
Comment (crypto3699) [inline .github/workflows/fortify.yml:34]: Payable

File: .github/workflows/fortify.yml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      contents: read
```
Comment (crypto3699) [inline .github/workflows/fortify.yml:35]: Payable

File: .github/workflows/fortify.yml [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+      security-events: write
```
Comment (crypto3699) [inline .github/workflows/fortify.yml:36]: Payable

File: .github/workflows/fortify.yml [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+    branches: [ "master" ]
+  schedule:
+    - cron: '38 1 * * 5'
+  workflow_dispatch:
+
+jobs:
+  Fortify-AST-Scan:
+    # Use the appropriate runner for building your source code. Ensure dev tools required to build your code are present and configured appropriately (MSBuild, Python, etc).
+    runs-on: ubuntu-latest
+    permissions:
```
Comment (crypto3699) [inline .github/workflows/fortify.yml:33]: Payable

**Review by crypto3699 — APPROVED** [review]

**Pull request closed without merging**